Palindromo.rb

```diff
@@ -0,0 +1,31 @@
+puts "Palindromo"
+
+class PalindromoExcersise
```

Comment:
is this a name for a class ? 
is this a name for a method?
what do you mean with piec?

let's review how a class should be defined and why

avoid mixin languages